jena-tdb/src/main/java/org/apache/jena/tdb/base/block/BlockMgrCache.java

```diff
@@ -58,10 +58,7 @@ private BlockMgrCache(int readSlots, int writeSlots, final BlockMgr blockMgr)
     {
```

Comment:
No problem. It was definitely to avoid `null`-checking, because `BlockMgrCache` sprouted all kinds of `null`-checks when I killed `Cache0`. As I commented there, I would like to rewrite `BlockMgrCache` to use `Optional` but I'm not yet confident enough that I understand it well enough, and I'm not yet familiar enough with the test framework to understand how well-covered `BlockMgrCache` is and how careful I need to be around it.